Source/UIKit/UITextField.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UITextField.swift
+//  Rex
+//
+//  Created by Rui Peres on 17/01/2016.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+import UIKit
+
+extension UITextField {
+    
+    /// Sends the field's string value whenever it changes.
+    public var rex_textSignal: SignalProducer<String, NoError> {
+        return NSNotificationCenter.defaultCenter()
+            .rac_notifications(UITextFieldTextDidChangeNotification, object: self)
```

Comment:
There's more to the retain cycle above than just `rex_pressed` since I've already got [tests around that](https://github.com/neilpa/Rex/blob/master/Tests/UIKit/UIButtonTests.swift#L41). I'm assuming that something with `viewModel.authenticate` is causing the problem.